src/gluonts/zebras/schema.py

```diff
@@ -29,27 +29,90 @@
 This module provides tooling to extract ``zebras.TimeFrame`` and
 ``zebras.SplitFrame`` instances from Python dictionaries::
 
-    schema = zebras.Schema(...)
+    schema = zebras.schema.Schema(...)
     tf = schema.load_timeframe(raw_dict)
 
     sf = schema.load_splitframe(raw_dict)
 
-The idea is to define expected types and shapes for each field (column) using
-``zebras.Field`` and then wrap them in a ``zebras.Schema``:
+The idea is to define expected types and shapes for each field (column) and
+then wrap them in a ``zebras.Schema``:
 
     schema = zebras.Schema({
-        "target": Field(ndim=1, tdim=0, past_only=True),
-        "time_feat": zebras.Field(ndim=2, tdim=-1),
-        "static_feat": zebras.Field(ndim=1),
+        "target": zebras.schema.TimeSeries(),
+        "time_feat": zebras.schema.TimeSeries(ndim=2, tdim=-1, past_only=False),
+        "static_feat": zebras.schema.Array(ndim=1),
```

Comment:
Could these just be keyword arguments to the constructor of `Schema`? So on can do it simpler:

```python
schema = Schema(
    target=Timeseries(),
    time_feat=Timeseries(...),
    static_feat=Array(...),
)
```

Reply:
I'm thinking to the case where we would use the schema simply for validation